src/frequenz/sdk/timeseries/ev_charger_pool/_ev_charger_pool.py

```diff
@@ -217,7 +217,6 @@ def power_distribution_results(self) -> ReceiverFetcher[_power_distributing.Resu
 
     async def stop(self) -> None:
         """Stop all tasks and channels owned by the EVChargerPool."""
-        await self._pool_ref_store.stop()
```

Comment:
Could you please add comment why this methods are empty?
It looks like bug now, because method description and definition does different thing.

Reply:
Done